.circleci/config.yml

```diff
@@ -0,0 +1,39 @@
+version: 2
```

Comment:
Is this the version of circle or the version number for our project?

Reply:
this is what the circle documentation says to put -- I think it's the version of circle, because everyone on the internet has it set to 2 (circle 2)